From 3475f3960cc6d7357dddbfb43945324f2faa6375 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Wed, 19 Jan 2022 07:44:08 -0700
Subject: [PATCH 3/5] Bump Security String to 2022-01-05

Implemented:
============
CVE:            References:  Type:  Severity:  Updated AOSP versions:
CVE-2021-0643   A-183612370  ID     High       10, 11, 12
CVE-2021-30353  A-202025599  RA     High
CVE-2021-39620  A-203847542  EoP    High       11, 12
CVE-2021-39621  A-185126319  EoP    High       9, 10, 11, 12
CVE-2021-39623  A-194105348  EoP    High       9, 10, 11, 12
CVE-2021-39626  A-194695497  EoP    High       9, 10, 11, 12
CVE-2021-39627  A-185126549  EoP    High       9, 10, 11, 12
CVE-2021-39628  A-189575031  ID     High       10, 11
CVE-2021-39629  A-197353344  EoP    High       9, 10, 11, 12
CVE-2021-39632  A-202159709  EoP    High       11, 12
CVE-2021-39659  A-208267659  DoS    High       10, 11, 12

Previously Implemented:
=======================
None

Not Implemented:
================
None

Not Applicable (platform source):
=================================
CVE:            References:  Type:  Severity:  Updated AOSP versions:
CVE-2020-0338   A-123700107  ID     High       9, 10
CVE-2021-39618  A-196855999  EoP    High       9, 10, 11, 12 (Pixel blobs)
CVE-2021-39622  A-192663648  EoP    High       10, 11, 12 (Pixel blobs)
CVE-2021-39625  A-194695347  EoP    High       9, 10, 11, 12 (Pixel blobs)
CVE-2021-39630  A-202768292  EoP    High       12

Change-Id: I9c1d84541a92e102dee926954a90710f1beb09b1
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index b5004e58b..9e1b85992 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -240,7 +240,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2021-12-05
+      PLATFORM_SECURITY_PATCH := 2022-01-05
 endif
 .KATI_READONLY := PLATFORM_SECURITY_PATCH
 
-- 
2.25.1

